src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -287,11 +287,15 @@ const buildWhereClause = ({ schema, query, index }): WhereClause => {
               inPatterns.push(`${listElem}`);
             }
           });
-          patterns.push(`(${name})::jsonb @> '[${inPatterns.join()}]'::jsonb`);
+          patterns.push(
+            `(${name})::jsonb @> '[${inPatterns.join(',')}]'::jsonb`
+          );
         } else if (fieldValue.$regex) {
           // Handle later
         } else {
-          patterns.push(`${name} = '${fieldValue}'`);
```

Comment:
Can you write a failing test with unsafe strings or strings pushed together `'"hello""world"'`?

I believe this could be written as
```
patterns.push(`$${index}:raw = '$${index + 1}'`);
values.push(name, fieldValue);
index += 2;
```

Actually this entire section could be rewritten.

What do you think?

Reply:
I think `name` will be like `"foo"->'bar'` so I don't know if a placeholder will work for that (?). Are there existing tests that run queries I can base my work on?

Reply:
It should work. :raw is basically replace but don’t format and don’t escape.

https://github.com/parse-community/parse-server/blob/master/spec/ParseQuery.spec.js#L362

If you need help running tests

https://github.com/parse-community/parse-server/blob/master/CONTRIBUTING.md

Reply:
@dplewis I don't have a query handy because I just noticed this in the code. If `fieldValue` contains a single quote, it should break.

Reply:
`query.equalTo('field', "this is a test's test")` would break?

Reply:
I think it would have to be a dot field like `query.equalTo('foo.bar', "In'valid")` but I've not tested this directly.

Reply:
Ok and what did you mean by `pushing strings together like '["hello""world"]'::jsonb.`

Reply:
@dplewis I had forgotten that the default separator of Array.join is `,`. So my changes on line 290 here are not necessary.

Reply:
I understand what you were trying to do.